src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DriverPanelControllerAdapter.java

```diff
@@ -30,7 +28,7 @@ public void operateTo(int x, int y) {
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
 
-		drawerController.drawLine(line);
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Dependency with DrawerFeature is not necessary. You can pass DrawPanelController instance in constructor.